lib/src/animations.dart

```diff
@@ -19,9 +19,15 @@ class CardSizes {
 
 /// Card Alignments
 class CardAlignments {
-  static Alignment front = Alignment(0.0, -0.5);
-  static Alignment middle = Alignment(0.0, 0.0);
```

Comment:
From what I see, you didn't add an 'option' to align the cards horizontally, you *replaced* the vertical alignment for horizontal. If you add an option for users to choose, I can consider merging this.